src/components/PageHero.tsx

```diff
@@ -118,11 +44,51 @@ const PageHero: React.FC<IProps> = ({
   const { buttons, title, header, subtitle, image, alt } = content
   return (
     <Content>
-      <HeroContainer isReverse={isReverse} className={className}>
-        <HeroContent>
-          <Title>{title}</Title>
-          <Header>{header}</Header>
-          <Subtitle>{subtitle}</Subtitle>
+      <Flex
+        justifyContent="space-between"
+        mt={8}
+        mb={0}
+        py={0}
```

Comment:
Do we need to specify these props? aren't they `0` anyway in the `Flex` component?

Reply:
You're right, Resolved.